cmd/lotus/daemon.go

```diff
@@ -269,7 +269,17 @@ var DaemonCmd = &cli.Command{
 			}
 		}
 
-		if cctx.Bool("remove-existing-chain") {
+		chainfile := cctx.String("import-chain")
+		snapshot := cctx.String("import-snapshot")
+		willImportChain := false
+		if chainfile != "" || snapshot != "" {
+			if chainfile != "" && snapshot != "" {
+				return fmt.Errorf("cannot specify both 'import-snapshot' and 'import-chain'")
+			}
+			willImportChain = true
+		}
+
+		if cctx.Bool("remove-existing-chain") || willImportChain {
```

Comment:
should we specify the behaviour expected if `remove-existing-chain` is set to be false yet `willImportChain` is true? warning & remove, or error out? or follow `remove-existing-chain`to not remove and continue with import?

Reply:
Good point, maybe we should just error out. Otherwise we are somehow creating undefined behaviors?

Reply:
@mb1896 What I would suggest is:

- if `remove-existing-chain`, just remove the chain
- if it's not specified, prompt the user with a y/n question asking whether they want the chain to be deleted -- if yes, do so, if not proceed without doing so

Reply:
Thansk for the comments! Fixed